packages/react-core/src/evm/hooks/wallet.ts

```diff
@@ -103,24 +103,24 @@ export function useChainId(): number | undefined {
  * Hook for accessing the active Chain the current wallet is connected to
  *
  * ```javascript
- * import { useActiveChain } from "@thirdweb-dev/react-core"
+ * import { useChain } from "@thirdweb-dev/react-core"
  * ```
  *
  * @example
  * You can get the chain of the connected wallet by using the hook as follows:
  * ```javascript
- * import { useActiveChain } from "@thirdweb-dev/react-core"
+ * import { useChain } from "@thirdweb-dev/react-core"
  *
  * const App = () => {
- *   const chain = useActiveChain()
+ *   const chain = useChain()
  *
  *   return <div>{chain.chainId}</div>
  * }
  * ```
  * @see {@link https://portal.thirdweb.com/react/react.useActiveChain?utm_source=sdk | Documentation}
  * @public
  */
-export function useActiveChain(): Chain | undefined {
+export function useChain(): Chain | undefined {
```

Comment:
Why this rename? This will return the active chain

Reply:
it's confusing - we have a different meaning for "activeChain" in thirdweb provider. This hook returns the chain that the wallet is connected to

Reply:
Discussed offline, we'll keep `useChain`. It returns the chain the wallet is connected to